packages/node/src/evm/handlers/fetch-pending-requests.test.ts

```diff
@@ -20,19 +20,20 @@ describe('fetchPendingRequests', () => {
       apiCalls: [
         {
           airnodeId: '0x19255a4ec31e89cea54d1f125db7536e874ab4a96b4d4f6438668b6bb10a6adb',
-          clientAddress: '0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0',
```

Comment:
Had to recreate all this because find and replace wasn't going to cut it this time